script/e2e-test.sh

```diff
@@ -271,12 +330,15 @@ pod=$(kubectl get po -l run=integration -o jsonpath="{.items[0].metadata.name}")
 for f in *.js; do
   kubectl cp "./${f}" "${pod}:/app/"
 done
+## Support for Docker registry secrets are not supported for Helm2, skipping that test
+if [[ "${HELM_VERSION:-}" =~ "v2" ]]; then
+  rm use-cases/create-private-registry.js
```

Comment:
We can't use https://jestjs.io/docs/en/configuration#testpathignorepatterns-arraystring ? No problem if it's a hassle to change.

Reply:
that will be a bit more elegant, yes